src/Microsoft.AspNetCore.Razor.VSCode/src/Html/HtmlProjectedDocumentContentProvider.ts

```diff
@@ -27,7 +27,9 @@ export class HtmlProjectedDocumentContentProvider implements vscode.TextDocument
             return;
         }
 
-        const content = razorDocument.htmlDocument.getContent();
+        const content = `${razorDocument.htmlDocument.getContent()}
+// ${razorDocument.htmlDocument.projectedDocumentSyncVersion}`;
```

Comment:
This is a bit quirky, seeing as `//` is not the syntax for comments in HTML. However I can't think of any problem that it's going to cause (since we only use the projected HTML doc for completions, etc., and the user isn't going to actually see the projected document rendered anywhere).

You could even eliminate the `//` if you wanted, as the effect would be the same.

Reply:
> You could even eliminate the `//` if you wanted, as the effect would be the same.

Can't because it'd result in the C# document being invalid. I ould technically do something different for Html but there's no harm in having `//` so for the sake of consistency I did the same for both virtual document.